CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,343 @@
+
+To better support situations where immediate action may be required, these guidelines are used *during* live events:
+
+* All conda-forge Organization events will have specific, named Code of Conduct contacts for the events.
+* The names and contact mechanisms for the Code of Conduct representatives will be clearly and frequently communicated to event participants.
+
+## CoC Representatives
+
+Every conda-forge Organization associated event will have named CoC Committee members or *CoC representatives* that are the first point of contact for that event. Who these people are will be clearly and frequently communicated to event participants. CoC representatives are used when there are no committee members participating in the event.
+
+## Live Events: Reporting and Actions
+
+At conda-forge Organization events, Code of Conduct committee members or representatives will attempt to gather and write down [information](#what-to-include-in-a-report) from anyone making a verbal report at a live event. Recording the details in writing is exceedingly important in order for us to effectively respond to reports. If event staff write down a report taken verbally, then the person making the report will be asked to review the written report for accuracy.
+
+For reports made during live events, or in any situation where urgent action is needed:
+
+* Any two (or more) event organizers, event staff, CoC Committee members or CoC representatives can decide if immediate action is to be taken and what that action is. In exceptionally dangerous situations, this decision can be made by a single person.
+* These rapid decisions can be reconsidered during the event as more information becomes available.
+* The scope of any rapid decision is limited to the current event / situation.
+* The report, any related information, and any decisions and consequences will be reported to the full Code of Conduct Committee as soon as possible.
+
+The full Code of Conduct Committee will then consider the report using the full timeline and processes defined below. The Committee may decide to apply consequences in other spaces beyond the space where the behavior was reported.
+
+Potential *immediate* consequences for violating the conda-forge Organization Code of Conduct at a live event include, but are not limited to:
+
+- Warning the person to cease their behavior and that any further reports will result in sanctions
+- Requiring that the person avoid any interaction with, and physical proximity to, the person they are harassing for the remainder of the event
+- Ending a talk that violates the policy early
+- Not publishing the video or slides of a talk that violated the policy
+- Not allowing a speaker who violated the policy to give (further) talks at the event now or in the future
```

Comment:
Tiny nitpick, but the "or in the future" clause here seems to conflict with the statement above that "[the] scope of any rapid decision is limited to the current event / situation"